<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialEditorWindow</class>
 <widget class="QWidget" name="MaterialEditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,2,0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="toolbarFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="editorFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTextEdit" name="scriptEditor"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="statusBarFrame">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="statusBar">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>213</red>
              <green>229</green>
              <blue>240</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>213</red>
              <green>229</green>
              <blue>240</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>213</red>
              <green>229</green>
              <blue>240</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>213</red>
              <green>229</green>
              <blue>240</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDockWidget" name="dockWidget">
     <widget class="QWidget" name="dockWidgetContents">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>845</width>
            <height>209</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTabWidget" name="tabWidget">
             <property name="currentIndex">
              <number>2</number>
             </property>
             <widget class="QWidget" name="RenderingOptionsTab">
              <attribute name="title">
               <string>Rendering</string>
              </attribute>
              <layout class="QFormLayout" name="formLayout">
               <property name="leftMargin">
                <number>9</number>
               </property>
               <property name="topMargin">
                <number>9</number>
               </property>
               <property name="rightMargin">
                <number>9</number>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="label_19">
                 <property name="text">
                  <string>Culling mode:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="cullingMode">
                 <item>
                  <property name="text">
                   <string>None</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Clockwise</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Counter clockwise</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_21">
                 <property name="text">
                  <string>Z Buffer:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QCheckBox" name="zBufferEnabled">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_20">
                 <property name="text">
                  <string>Z Buffer writing:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QCheckBox" name="zBufferWriteEnabled">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="TextureStageOptionsTab">
              <attribute name="title">
               <string>Texture stage</string>
              </attribute>
              <layout class="QFormLayout" name="formLayout_2">
               <property name="horizontalSpacing">
                <number>6</number>
               </property>
               <property name="verticalSpacing">
                <number>6</number>
               </property>
               <property name="margin">
                <number>9</number>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>U address:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="uAddressMode">
                 <item>
                  <property name="text">
                   <string>Wrap</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Mirror</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Clamp</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Border</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Mirror once</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_14">
                 <property name="text">
                  <string>V address:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="vAddressMode">
                 <item>
                  <property name="text">
                   <string>Wrap</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Mirror</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Clamp</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Border</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Mirror once</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_15">
                 <property name="text">
                  <string>W address</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QComboBox" name="wAddressMode">
                 <item>
                  <property name="text">
                   <string>Wrap</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Mirror</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Clamp</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Border</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Mirror once</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>Minification filter:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QComboBox" name="minFilter">
                 <item>
                  <property name="text">
                   <string>None</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Point</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Linear</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Anisotropic</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Pyramidal quad</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Gaussian quad</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_17">
                 <property name="text">
                  <string>Magification filter:</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QComboBox" name="magFilter">
                 <item>
                  <property name="text">
                   <string>None</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Point</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Linear</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Anisotropic</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Pyramidal quad</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Gaussian quad</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="label_18">
                 <property name="text">
                  <string>Mipmap filter:</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QComboBox" name="mipFilter">
                 <item>
                  <property name="text">
                   <string>None</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Point</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Linear</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Anisotropic</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Pyramidal quad</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Gaussian quad</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="OutputTab">
              <attribute name="title">
               <string>Output</string>
              </attribute>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
               <item>
                <widget class="QTextBrowser" name="compilationOutput"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
